soc/arm/st_stm32/stm32mp1/Kconfig.defconfig.series

```diff
@@ -40,4 +40,11 @@ config GPIO_STM32_PORTK
 
 endif # GPIO_STM32
 
+if I2C_STM32
```

Comment:
This change would better fit in previous commit.

Reply:
Done